openapi/webhooks/hard-usage-limit-reached.yaml

```
@@ -0,0 +1,11 @@
+summary: Soft usage limit reached
+operationId: soft-usage-limit-reached
+x-products:
+  - Core
+tags:
+  - Orders
+requestBody:
+  $ref: ../components/requestBodies/Webhooks/UsageLimitWebhook.yaml
+responses:
+  '2xx':
+    description: Returns any 2xx status to indicate that the data received.
```

suggestion: possible typo
```suggestion
    description: Returns any 2xx status to indicate that the is data received.
```
or 
```suggestion
    description: Returns any 2xx status to indicate that data received.
```